homeassistant/components/qvrpro/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "domain": "qvrpro",
```

Comment:
Thinking about it now, I would suggest to split this domain up. Please adjust other files, documentation accordingly.
```suggestion
  "domain": "qvr_pro",
```